pcap4j-core/src/main/java/org/pcap4j/packet/namednumber/DnsResourceRecordType.java

```diff
@@ -508,7 +508,7 @@
    * Certification Authority Restriction: 257
    */
   public static final DnsResourceRecordType CAA
```

Comment:
Please remove this change because IANA still defines 257 as Certification Authority Restriction.